src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -81,6 +81,7 @@
   --pf-c-dropdown__menu-item--disabled--Color: var(--pf-global--Color--dark-200);
   --pf-c-dropdown__menu-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
   --pf-c-dropdown__menu-item--disabled--BackgroundColor: transparent;
+  --pf-c-dropdown__menu-item--m-text--Color: var(--pf-global--Color--400);
```

Comment:
@mcarrano @maryshak1996 can we verify this is the correct color? This is the only place where we use `--pf-global--Color--400`. In the radio/check description, empty state body, and table caption, we use `--pf-global--Color--200`. Should we be using that instead?

Reply:
@mceledonia and I agreed that `--pf-global--Color--200` didn't provide enough contrast. 

@mcarrano @maryshak1996 wdyt?